src/app/views/interactives/InteractivesIndex.test.js

```diff
@@ -115,11 +151,12 @@ describe("Collections", () => {
         });
 
         it("should filter results when clicks apply button", () => {
+            useDispatchMock.mockReturnValue(filterInteractives);
             render(<InteractivesIndex {...defaultProps} />);
             const applyButton = screen.getByText("Apply");
             screen.getByLabelText("Internal ID").value = "Query";
             fireEvent.click(applyButton);
-            expect(defaultProps.filterInteractives.mock.calls).toHaveLength(1);
+            expect(filterInteractives).toHaveBeenCalled();
```

Comment:
can you assert the parameters here as well ?

Reply:
done